scripts/actor/actor-wfrp4e.js

```diff
@@ -3271,52 +3293,151 @@ class ActorWfrp4e extends Actor {
-     * choose the ones you do want.
-     *
-     * @param {Object} careerData     Career type Item to be used for advancement.
-     * 
-     * TODO Refactor for embedded entity along with the helper functions
-     */
-    async _advanceNPC(careerData) 
-    {
-      let updateObj = {};
-      let advancesNeeded = careerData.level.value * 5; // Tier 1 needs 5, 2 needs 10, 3 needs 15, 4 needs 20 in all characteristics and skills
-  
-      // Update all necessary characteristics to the advancesNeeded
-      for (let advChar of careerData.characteristics)
-        if (this.data.data.characteristics[advChar].advances < 5 * careerData.level.value)
-          updateObj[`data.characteristics.${advChar}.advances`] = 5 * careerData.level.value;
-  
-      // Advance all skills in the career
-      for (let skill of careerData.skills)
-        await this._advanceSkill(skill, advancesNeeded);
-  
-      // Add all talents in the career
-      for (let talent of careerData.talents)
-        await this._advanceTalent(talent);
-  
-      this.update(updateObj);
-    }
+  /**
+   * Advance NPC based on given career
+   * 
+   * A specialized function used by NPC type Actors that triggers when you click on a 
+   * career to be "complete". This takes all the career data and uses it (and the helpers
+   * defined above) to advance the actor accordingly. It adds all skills (advanced to the 
+   * correct amount to be considered complete), advances all characteristics similarly, and 
+   * adds all talents.
+   * 
+   * Note: This adds *all* skills and talents, which is not necessary to be considered complete.
+   * However, I find deleting the ones you don't want to be much easier than trying to pick and 
+   * choose the ones you do want.
+   *
+   * @param {Object} careerData     Career type Item to be used for advancement.
+   * 
+   * TODO Refactor for embedded entity along with the helper functions
+   */
+  async _advanceNPC(careerData) 
+  {
+    let updateObj = {};
+    let advancesNeeded = careerData.level.value * 5; // Tier 1 needs 5, 2 needs 10, 3 needs 15, 4 needs 20 in all characteristics and skills
 
+    // Update all necessary characteristics to the advancesNeeded
+    for (let advChar of careerData.characteristics)
+      if (this.data.data.characteristics[advChar].advances < 5 * careerData.level.value)
+        updateObj[`data.characteristics.${advChar}.advances`] = 5 * careerData.level.value;
 
-    _replaceData(formula) {
-      let dataRgx = new RegExp(/@([a-z.0-9]+)/gi);
-      return formula.replace(dataRgx, (match, term) => {
-        let value = getProperty(this.data, term);
-        return value ? String(value).trim() : "0";
-      });
+    // Advance all skills in the career
+    for (let skill of careerData.skills)
+      await this._advanceSkill(skill, advancesNeeded);
+
+    // Add all talents in the career
+    for (let talent of careerData.talents)
+      await this._advanceTalent(talent);
+
+    this.update(updateObj);
+  }
```

Comment:
What was changed here? Anything?

Reply:
Hmm no, nothing. Must be a white space change, you can ignore them in the review settings